src/ol_infrastructure/infrastructure/vault/pki/__main__.py

```diff
@@ -0,0 +1,84 @@
+from pulumi import Config, export
+
+from ol_infrastructure.components.services.vault import (
+    OLVaultPKIIntermediateCABackend,
+    OLVaultPKIIntermediateCABackendConfig,
+    OLVaultPKIIntermediateEnvBackend,
+    OLVaultPKIIntermediateEnvBackendConfig,
+    OLVaultPKIIntermediateRole,
+    OLVaultPKIIntermediateRoleConfig,
+)
+from ol_infrastructure.lib.ol_types import BusinessUnit
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+from ol_infrastructure.lib.vault import VaultPKIKeyTypeBits
+
+SIX_MONTHS = 60 * 60 * 24 * 30 * 6  # noqa: WPS432
+TWELVE_MONTHS = 60 * 60 * 24 * 30 * 12  # noqa: WPS432
+
+env_config = Config("environment")
+stack_info = parse_stack()
+business_unit = env_config.get("business_unit") or "operations"
+
+# The files should be generated by the offline Root CA and copied locally prior to
+# running this. More information here - https://docs.odl.mit.edu/en/platform/security/vault-pki
+try:
+    with open("/tmp/int_ca_pe_bundle_file.pem") as int_ca_pem_bundle_file, open(
+        "/tmp/int_ca_cert_file"
+    ) as int_ca_cert_file:
+        pki_intermediate_ca_config = OLVaultPKIIntermediateCABackendConfig(
+            max_ttl=TWELVE_MONTHS,
+            default_ttl=TWELVE_MONTHS,
+            intermediate_ca_pem_bundle=int_ca_pem_bundle_file,
```

Comment:
Shouldn't this be doing a `.read()` to pass the contents of the file rather than just the file object?